### PR TITLE
Syndicate Tools Rework

### PR DESCRIPTION
**Pull request by Steelpoint**
**DO NOT MERGE**: PR contains antagonist only items, as such it cannot be merged. I am opening this mainly for feedback.

-----

This PR effects several changes to the tools a Traitor or Nuclear Operative can purchase.

synditools

* New sprites! These sprites currently only appear on the Nuke Ops ship and if a Nuke Op purchases Syndi Tools. Standard Traitors get the 'stealth' version of these tools, meaning they use standard station sprites.
* Syndi Multitool functions akin to a Abductor Multitool. Meaning it will display the function of every single wire to the user. Makes the tool far more useful.
* All Syndicate tools work twice as fast as their regular NT variants. 

My main point of contention is if a Traitor that buys a Toolbox should get the Syndi sprites or they should get more stealthy tools that use standard sprites, as currently only Ops get the new sprites.

**NOTE**: All Syndicate Toolboxes that spawn on ruins or anywhere similar, aside from the Nuke Ops ship, do NOT get the special syndicate tools. They will continue to use the standard tools. 

:cl: Steelpoint
add: The Syndicate have begun issuing field operatives with advanced engineering tools within syndicate toolboxes, which work twice as fast. This includes a prototype multitool that displays the wire functions of whatever it is examining.
tweak: Covert Syndicate operatives get disguised engineering tools if they purchase a suspicious syndicate toolbox. Nuclear Operatives do not get a disguised toolset if they purchase a toolbox.
/:cl:

### CODE REVIEW

**Comment by lzimann** [comment]
Freeze and all of that

**Comment by Anonmare** [comment]
I like it, though I may be biased because I suggested it

**Comment by Steelpoint** [comment]
It would actually make the toolset more worthwhile in buying, even then its still more of an emergency purchase or something you do with a long term goal in mind (breaking into armoury via getting arrested and hiding tools in a internal implant for example).

**Comment by Anonmare** [comment]
I'm hoping it would make the syndi toolbox more attractive due to the superior tools. I'm hoping the multitool will, at the very least, give pause for thought due to it being able to see what each wire does at a glance. 
Potentially even giving people ideas for sabotage that would be too time consuming to find out manually (such as electrifying an airlock then putting another airlock in front of it, before messing with the safeties and door timer on the second door to door slam someone to death).

**Comment by Steelpoint** [comment]
Only thing I want to know is if the traitor issued items should have the syndie sprites or standard sprites.

**Comment by Anonmare** [comment]
Personally I'd say standard sprites but it's your call since the syndi-sprites are pretty good looking

File: code/game/objects/items/devices/multitool.dm [hunk 1]
```diff
@@ -96,4 +96,17 @@
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
-	origin_tech = "magnets=5;engineering=5;abductor=3"
\ No newline at end of file
+	origin_tech = "magnets=5;engineering=5;abductor=3"
+
+/obj/item/device/multitool/abductor/syndi
```
Comment (GunHog) [inline code/game/objects/items/devices/multitool.dm]: Steelpoint. No. I told you the right way to do this! Remove the type check in the line I provided on the forums, replace with a var!

Comment (Steelpoint) [inline code/game/objects/items/devices/multitool.dm]: Beyond my abilties so I can't do it

Comment (GunHog) [inline code/game/objects/items/devices/multitool.dm]: But...I told you exactly what to do...were my instructions unclear or otherwise poorly written?

Comment (Steelpoint) [inline code/game/objects/items/devices/multitool.dm]: You said to check for a var but I don't know how to do that.

Again, beyond my abilties.

Comment (GunHog) [inline code/game/objects/items/devices/multitool.dm]: https://tgstation13.org/phpBB/viewtopic.php?f=9&t=30&p=313905#p313831

I know your PR history, although you make mostly sprite, map, and numerical changes, you have done things a bit more challenging code-wise as well. This is not beyond your ability, you simply have not given it a shot yet.

Comment (Steelpoint) [inline code/game/objects/items/devices/multitool.dm]: It is. everything else I've done is either a numerical change or copy+pasting from elsewhere

**Comment by Armhulen** [comment]
make the traitors use syndi sprites considering these are twice as fast and the multitool sees every pulse that's extremely strong

**Comment by Steelpoint** [comment]
I was leaning towards making the traitors also use the syndie sprites.

**Comment by Steelpoint** [comment]
Requesting this be reopened.

**Comment by Anonmare** [comment]
Kor will probably tell you to open another PR yourself

He did for me

**Comment by Steelpoint** [comment]
@KorPhaeron asking for this to be opened please.

**Comment by GunHog** [comment]
Just make a new PR!

**Comment by KorPhaeron** [comment]
Make a new one so it doesnt appear at the back of the PR list

**Pull request closed without merging**